documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_2_Identity_and_Trust_Management.md

```diff
@@ -0,0 +1,113 @@
+  * If connector is run in one protected VM (e.g. SEV SNP): 1 P_UID is mapped to 1 PIK for this SEV-SNP VM.
+  * If multiple protected VMs (e.g. SEV SNP) form a distributed platform setup: 1 UID for the setup maps to n PIKs for the VMs which comprise this setup.
+
+![Identity mapping for different scenarios](./media/identity_mapping.png)
+#### _Figure 4.1.2.2: Identities for IDS Connector Services and Platforms_
+(Remark: The platforms in the image may always be either physical devices or protected VMs)
+
+### Describing Metadata
+The IDS targets not only a secure exchange of data but also a trustworthy environment for data processing honoring the defined usage control policies. To achieve this goal, it is not sufficient to only know the identity of another IDS component, but additional information about the company operating the component and the utilized software stack is required.
+This information is provided in form of the following describing artifacts:
+* A **Company Description** for each company operating an IDS component which contains verified information about the company as well as information about its Operational Environment Certification (explained in [Chapter 4.2.3](../4_2_Certification_Perspective/4_2_3_Operational_Environment_Certification.md)).
+* **Software Manifests** for the utilized software components which have been evaluated in the Component Certification explained in [Chapter 4.2.4](../4_2_Certification_Perspective/4_2_4_Component_Certification.md)). In addition to the awarded certification levels, the manifests contain verified measurements which can be used to validate that the described software is truly running on the device. To support re-usability of components, the description of each software stack consists of three types of Software Manifests used for describing different layers:
+  * A **Root of Trust for Measurement (RTM) Manifest** for components of the boot stage,
+  * an **Operating System (OS) Manifest** for utilized kernel and user space components, including the container run time enabling the execution of different isolated containers/apps, and
+  * an arbitrary number of **App Manifests** per component identifying the utilized containers/apps.
+Signatures are utilized to represent the passed stages of the certification process (described in [Chapter 4.2.5](../4_2_Certification_Perspective/4_2_5_Processes.md)) and allow a validation of the correctness of the describing artifacts.
+In addition to these static artifacts, the connector operator may add additional attributes to the component's description or have them validated and registered at the DAPS.
+
+### Interactions between IDS Connectors and Identity Components
+
+To establish a trusted connection, each connector needs the identity information of the corresponding connector to perform access and usage control decisions. The interactions can be depicted as follows:
```

Comment:
How does this integrate into the ParIS concept?

Reply:
Added this. ParIS so far does only serve untrusted information and is not involved into trust building. Before working further on ParIS, we need a new driving person for this topic.